charts/memgraph/values.yaml

```diff
@@ -4,27 +4,71 @@ image:
   tag: ""
   pullPolicy: IfNotPresent
 
-# Can be increased to more but there is no replication or HA support in this chart.
+## Image Secrets for custom images
```

Comment:
Can we use booleans for configs? This way the control is a bit clearer, here is an example
```
# values.yaml
useImagePullSecrets: false
imagePullSecrets:
  - name: regcred
```
```
spec:
  {{- if .Values.useImagePullSecrets }}
  imagePullSecrets:
    {{- toYaml .Values.imagePullSecrets | nindent 4 }}
  {{- end }}

```

Reply:
Yes or you can just momment and format as you proposed:
```
useImagePullSecrets: false
imagePullSecrets:
  - name: regcred
 ```